src/lib/modules/job/job.go

```diff
@@ -0,0 +1,182 @@
+package job
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"sync"
+	"time"
+
+	config "github.com/bench-routes/bench-routes/src/lib/config_v2"
+	"github.com/bench-routes/bench-routes/src/lib/modules/executor"
+	"github.com/bench-routes/bench-routes/tsdb/file"
+)
+
+// Executable is an interface that is implemented by machineJob and monitoringJob
+type Executable interface {
+	Execute()
+	Abort()
+	Info() *JobInfo
+}
```

Comment:
```suggestion
// Executable is an interface that is implemented by machineJob and monitoringJob.
type Executable interface {
	Execute()
	Abort()
	JobInfo() *JobInfo
}
```

Reply:
I guess this will cause an error since any Executable, whether machineJob or monitorJob, already has field JobInfo, it can't have a function with the same name.